docs/conf.py

```diff
@@ -46,6 +46,7 @@
     "mergedeep",
 ]
 napoleon_numpy_docstring = True
+pygments_style = "sphinx"
```

Comment:
Why are there two `conf.py` files?  Will this file ever be used by Sphinx?

Reply:
Best I can tell, no that file doesn't ever get used. I copied this cookiecutter from another repository, and I suspect this is cruft.

I've opened an issue for it: #10